modules/setup/setup.js

```diff
@@ -243,21 +243,13 @@ class Setup {
             {
                 name: `${CONTAINER_NAME}`,
                 ExposedPorts: {
-                    '18731/tcp': {},
-                    '18732/tcp': {},
-                    '18733/tcp': {},
+                    '18731/tcp': {}
```

Comment:
why are we still exposng 18731  instead of 18732?

Reply:
We are mapping docker port no. 18731 with host port 18732. I think it must expose docker port 18731.